api/maven-api-settings/pom.xml

```diff
@@ -28,8 +28,8 @@ under the License.
 
   <artifactId>maven-api-settings</artifactId>
 
-  <name>Maven API Settings</name>
-  <description>Maven API Settings model.</description>
+  <name>Maven 4 API - Settings</name>
```

Comment:
So this module contains settings or the settings model? This is a difference.

Reply:
I don't get

Reply:
Settings is something you configure/set on something, a settings module is the transport class which is used to plug the actual settings between a user and the backend.

Looking at the code it looks like Maven 4 API :: Settings Utils/Helpers. I don't see any settings nor a model.

Reply:
The jar includes the generated model too:
  https://maven.apache.org/ref/4-LATEST/apidocs/org/apache/maven/api/settings/package-summary.html